static/js/narrow_banner.js

```diff
@@ -118,15 +119,15 @@ function pick_empty_narrow_banner() {
             switch (first_operand) {
                 case "starred":
                     // You have no starred messages.
-                    return $("#empty_star_narrow_message");
+                    return "empty_star_narrow_message";
```

Comment:
After looking at this code, I think it might be a lot more readable to remove the indirection and inline the content you have in the other file here, i.e. having this be something like this:

```
return {
        title: $t({defaultMessage: "You haven't starred anything yet!"}),
        html: $t_html({
            defaultMessage: "Learn more about starring messages <z-link>here</z-link>"
        },{
            "z-link": (content_html) => `<a href="/help/star-a-message">${content_html}</a>`
        })
}
```

That way, one doesn't need to look at multiple files to figure out what's going to happen.